app/src/org/commcare/android/view/GraphView.java

```diff
@@ -184,7 +184,11 @@ private void renderSeries(SeriesData s) {
         for (XYPointData p : sortedPoints) {
             if (mData.getType().equals(Graph.TYPE_BUBBLE)) {
                 BubblePointData b = (BubblePointData) p;
-                ((RangeXYValueSeries) series).add(parseXValue(b.getX()), parseYValue(b.getY()), parseRadiusValue(b.getRadius()));
+                try {
+                    ((RangeXYValueSeries) series).add(parseXValue(b.getX()), parseYValue(b.getY()), parseRadiusValue(b.getRadius()));
+                } catch(NumberFormatException nfe){
+                    System.out.println("Number format exception: " + nfe.toString());

```

Comment:
(didn't mean to include this in this branch, is from my internal debugging. closing)